SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/WorkerRegion.cpp

```diff
@@ -47,7 +47,7 @@ void AWorkerRegion::SetOpacity(const float Opacity)
 	MaterialInstance->SetScalarParameterValue(WORKER_REGION_MATERIAL_OPACITY_PARAM, Opacity);
 }
 
-void AWorkerRegion::SetExtents(const FBox2D& Extents)
+void AWorkerRegion::SetPositionAndScale(const FBox2D& Extents, const float VerticalScale)
```

Comment:
This will spawn the regions centred on the ground and will extend downwards? Not sure if this was intentional. It's probably the right thing to do imo since for anything below 0 on z axis, i.e. a cave or something, regions would disappear

Reply:
Yep it will scale up and down, centered on CurrentLocation.Z, which is set by SetHeight.